modules/@apostrophecms/schema/ui/apos/logic/AposInputArray.js

```diff
@@ -124,16 +175,66 @@ export default {
     }
   },
   methods: {
+    toggleAll(open) {
+      this.items = this.items.map(item => ({
+        ...item,
+        open
+      }));
+    },
+    startDragging(event) {
+      this.isDragging = true;
+      this.disengageAll();
+      this.toggleEngage({ target: event.item });
+    },
+    stopDragging(event) {
+      this.isDragging = false;
+      document.getSelection().removeAllRanges();
+      this.focusElement(event.item.getAttribute('data-id'));
+    },
+    getInlineMenuItems(index) {
+      const menu = klona(this.inlineContextMenu);
+      if (index === 0 && menu.find(i => i.action === 'move-up')) {
```

Comment:
Detail but usins `some` on array would return boolean:
```javascript
      if (index === 0 && menu.some(i => i.action === 'move-up')) {
```
A little more appropriate to if statements.